src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```
@@ -335,6 +335,22 @@ FINANCIAL_REPORTS:
     ROOT_PATH: sandbox
     STORAGE_TYPE: localfs
 FOOTER_ORGANIZATION_IMAGE: /images/logo.png
+FORUM_V2_ENABLE_MYSQL_BACKEND: true
+FORUM_SEARCH_BACKEND: "forum.search.es.ElasticsearchBackend"
+
+FORUM_ELASTIC_SEARCH_CONFIG:
+  - host: "{{ keyOrDefault \"elasticsearch/host\" \"elasticsearch.service.consul\" }}"
+    port: "443"
+    use_ssl: true
+
+FORUM_ELASTIC_SEARCH_CONFIG_ES7:
+  - host: "{{ keyOrDefault \"elasticsearch/host\" \"elasticsearch.service.consul\" }}"
+    port: "443"
+    use_ssl: true
+
+FORUM_MONGODB_CLIENT_PARAMETERS: mongo_params
+
+FORUM_MONGODB_DATABASE: "edxapp"
```

```suggestion
FORUM_MONGODB_DATABASE: "forum"
```
This needs to point at the [existing Mongo database entry](https://github.com/mitodl/ol-infrastructure/blob/main/src/bilder/images/forum/files/forum.env.j2#L2C138-L2C143) that the cs_comments_service forum is using. That way it provides continuity of content between the two implementations, as well as allowing us to point at the pre-existing content for eventual migration into the MySQL database.